samples/idea-latest.yaml

```diff
@@ -0,0 +1,36 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: idea-latest
+spec:
+  started: false
```

Comment:
Should be 
```suggestion
  started: true
```
to avoid having to patch the workspace to start it

Reply:
Yes, it was `true` before, but then changed it to test the dashboard and understand why I meet the error
![Screenshot from 2023-02-01 19-12-47](https://user-images.githubusercontent.com/1655894/216276519-b4f71240-ee0c-420f-910a-3e25c03f69f7.png)